components/brave_new_tab_ui/components/default/widgetTitleTab/index.ts

```diff
@@ -6,6 +6,22 @@ import styled from 'styled-components'
 
 interface StyleProps {
   isAlone?: boolean
+  stackPosition: number
+}
+
+const getBackgroundRule = (position: number) => {
```

Comment:
I originally wrote something to dynamically generate these gradient rules based on the position, but this is more explicit and readable